src/9.negotiation.js

```diff
@@ -20,6 +20,20 @@ import {
 
 const expUnicodeExSeq = /-u(?:-[0-9a-z]{2,8})+/gi; // See `extension` below
 
+defineProperty(Intl, 'getCanonicalLocales', {
```

Comment:
In the spec, it belongs to the Intl section, which was expanded to support new members.